main.js

```diff
@@ -88,28 +68,55 @@ class Timer {
 }
 
 class DOM_Manager {
-    constructor() {
-        this.cardsCountVar = document.getElementsByClassName("card").length;
+    constructor(Fn) {
+        this.callBackFn=Fn;
         this.clickListenerBinded = this.clickListener.bind(this);
+        this.submitOnClickBinded = this.submitOnClick.bind(this);
         this.imgs = [];
-        for(let i=0;i<this.cardsCountVar;i++){
-            this.imgs.push(document.getElementById("img"+i));
-        }
+        this.imagesRandomised=[];
+        document.getElementById("Easy").addEventListener("change", this.difficultOnChange);

```

Comment:
I think it's better to split `DOM_Manager` into 2 classes:
- One for cards manipulation
- One for initial form manipulation

One class should be responsible for only one thing. It would be hard to remove or reuse level selection logic otherwise